safe-global/safe-wallet-monorepo
# Fix(Outreach): use versioning for outreach popup local storage flags [SW-690]

## What it solves

Resolves: https://www.notion.so/safe-global/76f265123c0a4ad680947ce489804342?v=f9f164524ec4464286fdc38f31589d0c&p=1628180fe573802f8b60f57c436d89da&pm=s

## How this PR fixes it
- adds versions to the localStorage and session storage flags used by the outreach popup

## How to test it
- Open a safe in a browser where the outreach popup was dismissed during the previous campaign. You can simulate this by creating a variable in localStorage called `SAFE_v2__outreachPopup` and setting it to true.
- Open any safe as a signer. The new campaign outreach popup should still be shown.


## Checklist
